demo/demo.cpp

```diff
@@ -16,7 +16,7 @@
 #include "model_loader_assimp.hpp"
 #include "d3d12/d3d12_dynamic_descriptor_heap.hpp"
 
-#define SCENE viknell_scene
```

Comment:
Revert this. We want the same scene to be default unless we find a better scene. Makes benchmarking older versions easier.